### PR TITLE
rm grunt, upgrade pegjs

### PR DESCRIPTION
**Pull request by longlho**
Grunt is `0xdeadbeef`
pegjs also released a new major version & grunt-peg is unmaintained. pegjs@0.11 supports `--format=es` so we can get rid of the `export default ${parser}` hack.

I tested in `intl-messageformat` and things seem fine.

### CODE REVIEW

**Comment by longlho** [comment]
ping @redonkulus

**Comment by redonkulus** [comment]
Any breaking changes or is this a minor release?

**Comment by longlho** [comment]
major just to be safe?

**Comment by longlho** [comment]
the only potential breaking change is `pegjs` version bump, but I tested in `intl-messageformat` and everything seems fine

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-- 8
```
Comment (okuryu) [inline .travis.yml]: why remove 8? it is still LTS version.

**Comment by longlho** [comment]
Sorry that was my bad. The monorepo should have the correct travis support
which we’re migrating to

**Pull request merged**